interface/ceed-fortran.c

```diff
@@ -438,53 +438,37 @@ static int CeedQFunction_count = 0;
 static int CeedQFunction_n = 0;
 static int CeedQFunction_count_max = 0;
 
-struct fContext {
-  void (*f)(void *ctx, int *nq,
-            const CeedScalar *u,const CeedScalar *u1,const CeedScalar *u2,
-            const CeedScalar *u3,
-            const CeedScalar *u4,const CeedScalar *u5,const CeedScalar *u6,
-            const CeedScalar *u7,
-            const CeedScalar *u8,const CeedScalar *u9,const CeedScalar *u10,
-            const CeedScalar *u11,
-            const CeedScalar *u12,const CeedScalar *u13,const CeedScalar *u14,
-            const CeedScalar *u15,
-            CeedScalar *v,CeedScalar *v1, CeedScalar *v2,CeedScalar *v3,
-            CeedScalar *v4,CeedScalar *v5, CeedScalar *v6,CeedScalar *v7,
-            CeedScalar *v8,CeedScalar *v9, CeedScalar *v10,CeedScalar *v11,
-            CeedScalar *v12,CeedScalar *v13, CeedScalar *v14,CeedScalar *v15, int *err);
-  void *innerctx;
-};
-
 static int CeedQFunctionFortranStub(void *ctx, int nq,
                                     const CeedScalar *const *u, CeedScalar *const *v) {
-  struct fContext *fctx = ctx;
+  fContext *fctx = ctx;
   int ierr;
 
-  CeedScalar ctx_=1.0;
```

Comment:
Previously we were simply suppling the user QFunction with false data for the ctx.